src/Sarif.UnitTests/ExtensionsTests.cs

```diff
@@ -38,31 +38,31 @@ public void Extensions_PropertValue_SetValue()
         [TestMethod]
         public void Extensions_IsNewline_CarriageReturn()
         {
-            Assert.IsTrue(Extensions.IsNewline('\r'));
+            Assert.IsTrue(ExtensionMethods.IsNewline('\r'));
```

Comment:
>Assert.IsTrue [](start = 12, length = 13)

Since you're in this file, let's take the opportunity to introduce FluentAssertions (just the file you're touching).

Reply:
```
'\r'.IsNewline().Should().BeTrue();
```

How nice is that?

---
In reply to: [136153377](https://github.com/Microsoft/sarif-sdk/pull/612#discussion_r136153377) [](ancestors = 136153377)